service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapperTest.java

```diff
@@ -36,92 +39,109 @@
+        "observation_compound_statement_with_interpretation_code_abnormal.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_DATA_ABSENT_REASON_AND_INTERPRETATION_AND_BODY_SITE_AND_METHOD_XML =
-        OBSERVATION_TEST_FILE_DIRECTORY
-        + "observation_compound_statement_with_data_absent_reason_and_interpretation_and_body_site_and_method.xml";
+        "observation_compound_statement_with_data_absent_reason_and_interpretation_and_body_site_and_method.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_VALUE_QUANTITY_AND_REFERENCE_RANGE_XML =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_with_value_quantity_and_reference_range.xml";
+        "observation_compound_statement_with_value_quantity_and_reference_range.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_REFERENCE_RANGE_AND_INTERPRETATION_XML =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_with_reference_range_and_interpretation.xml";
+        "observation_compound_statement_with_reference_range_and_interpretation.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_AVAILABILITYTIME_AND_LOW_EFFECTIVETIME_XML =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_with_availabilitytime_and_low_effectivetime.xml";
+        "observation_compound_statement_with_availabilitytime_and_low_effectivetime.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_NO_EFFECTIVE_PROPERLY_HANDLED =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_with_no_effective_properly_handled.xml";
+        "observation_compound_statement_with_no_effective_properly_handled.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_REFERENCERANGE_IN_COMMENT =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_with_referencerange_in_comment.xml";
+        "observation_compound_statement_with_referencerange_in_comment.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_WITH_VALUE_STRING_OUTPUT =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_with_value_string.xml";
+        "observation_compound_statement_with_value_string.xml";
     private static final String OBSERVATION_COMPOUND_STATEMENT_CLUSTERED_BY_DIAGNOSTICREPORT_REFERENCE_XML =
-        OBSERVATION_TEST_FILE_DIRECTORY + "observation_compound_statement_clustered_by_diagnosticreport_reference.xml";
-    private static final String OBSERVATION_COMPOUND_STATEMENT_1_WITH_RELATED_COMMENT_XML = OBSERVATION_TEST_FILE_DIRECTORY
-        + "observation_compound_statement_1_with_related_comment.xml";
+        "observation_compound_statement_clustered_by_diagnosticreport_reference.xml";
+    private static final String OBSERVATION_COMPOUND_STATEMENT_1_WITH_RELATED_COMMENT_XML =
+        "observation_compound_statement_1_with_related_comment.xml";
```

Comment:
I think these variables can be put in one line instead of splitting them in two lines of code, it will be more concise.

Reply:
Hey @ole4ryb, 

The reason for splitting these were due to check style constraints - I think keeping these consistent could be
good - but happy to hear your opinions! 😄